spec/features/user_sign_up_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.feature 'User sign ups', type: :feature do
+  let!(:user) { create(:user) }
+
+  background do
+    visit root_path
+    click_on 'Sign up'
+  end
+
+  scenario 'with valid values' do
+    fill_in 'Email', with: 'jvargas18@gmail.com'
+    fill_in 'Password', with: '123456'
+    fill_in 'Password confirmation', with: '123456'
```

Comment:
Maybe you just haven't realized, but it would be better to avoiding hardcoded values, something like: 
```suggestion
    valid_user = build(:user)
    fill_in 'Email', with: valid_user.email
    fill_in 'Password', with: valid_user.password
    fill_in 'Password confirmation', with: valid_user.password
```